MvvmCross/Platform/Ios/Core/MvxApplicationDelegate.cs

```diff
@@ -63,5 +61,30 @@ private void FireLifetimeChanged(MvxLifetimeEvent which)
         }
 
         public event EventHandler<MvxLifetimeEventArgs> LifetimeChanged;
+
+        protected virtual void CreateSetup(IMvxApplicationDelegate applicationDelegate, UIWindow window)
+        {
+            var setupType = FindSetupType();
```

Comment:
We should not be doing this - reflection is slow. We can have this as a fallback but the need the ability for the setup class to be specified (see UWP)

Reply:
This is outdated code. Look at the whole PR.